src/VncClient.cpp

```diff
@@ -119,7 +124,9 @@ namespace
 
 namespace Rfb
 {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 8, 0)
```

Comment:
moc is only for the enum to string conversion used for debugging. For early Qt versions, where moc is not processing namespaces, we have 2 options:

- debug the enums as ints
- make the namespace a dummy class RfbData being derived from QObject

Reply:
I made a RfbData class.